src/components/content.js

```diff
@@ -659,7 +659,7 @@ class Content extends React.Component {
       const anchorInline = document.getClosestInline(anchor.key)
       const focusInline = document.getClosestInline(focus.key)
 
-      if (anchorInline && !anchorInline.isVoid && anchor.offset == anchorText.length) {
+      if (anchorInline && anchorInline.isVoid && anchor.offset == anchorText.length) {
```

Comment:
Nah, i was wrong. In some cases it is not working like expected.

What i would like to fix here is to place the selection __directly__ after the inline node and not after the next text node